lib/hanami/cli/commands/app/dev.rb

```diff
@@ -45,10 +35,8 @@ def call(procfile: nil, **)
 
           # @since 2.1.0
           # @api private
-          def executable(procfile: nil)
-            # TODO: support other implementations of Foreman
-            # See: https://github.com/ddollar/foreman#ports
-            ["foreman", ["start", "-f", procfile || "Procfile.dev"]]
+          def executable
+            ["bin/dev"]
```

Comment:
```suggestion
            [::File.join("bin", "dev")]
```

Reply:
Applied this, thanks @jodosha.

Even though `File.join`—despite giving the appearances of being a cross-platform solution— is basically is a glorified "/"-joiner 🙃 

https://github.com/ruby/ruby/blob/f6e2f32a962c2c19a0418dfd2020c8c9ab32c8b0/file.c#L5064